### ISSUE TITLE
Accessibility: 

### ISSUE THREAD

**Issue by Abolade017** [jupyterhub/outreachy#91]
### What page is this for?

https://github.com/jupyterhub/outreachy/issues/38#issuecomment-1277526229

### WAVE accessibility report

I have evaluated the web accessibility of [https://jupyter.org/governance/governance.html](url) page. The evaluation done stated the following issues and how to fix those issues are below:

![governance](https://user-images.githubusercontent.com/70269933/195594583-60315391-7eba-4097-9acf-10380118655e.JPG)

## **Evaluation**

1. **9 Errors** 
- **Empty form label (5) :** A form label is present and associated with an existing form control (using for/id or surrounds the form control), but does not contain any text or images with alternative text. This issue can be fixed by ensuring that the form label contains text that describes the function of the associated form control. Labels are not required for image, submit, reset, button, or hidden form controls. If a label is not necessary visually, a descriptive title attribute may be added to the form control.
- **Multiple form labels (2) :** Two or more `<label>s` are associated to a single `<input>` (except types of image, submit, reset, button, or hidden), `<select>`, or `<textarea>`. This issue can be fixed by ensuring that at most one label element is associated to the form control. If multiple form labels are necessary, use `aria-labelledby`.
- **Language missing or invalid (1) :** The `<html lang>` attribute is missing or is empty, or a `lang` attribute value is not a valid language identifier. This issue can be fixed by identifying the document language using the `<html lang>` attribute with a valid value (e.g., `<html lang="en">`). Ensure that all `lang` attribute values are valid. 
- **Empty button (1) :** A <button> element is present that contains no text content (or alternative text), or an `<input type="submit">`, `<input type="button">`, or `<input type="reset">` has an empty or missing value attribute. This issue can be fixed by placing text content within the `<button>` element or give the `<input>` element a value attribute.

2. **3 Features**
- **Linked image with alternative text (1) :** An image element has non-empty alternative text, is within a link, and no other text (or images with alternative text) is present within the link. This issue can be fixed by ensuring that the alternative text presents the content of the image and/or the function of the link. If the full content and function of the link is presented in text within the link (an image and a text caption both within the same link, for example), then the image should generally be given empty/null alternative text (`alt=""`) to avoid redundancy.
- **Form label (2) :** A `<label>` element is present and properly associated to `<input>` (except types of image, submit, reset, button, or hidden), `<textarea>`, or `<select>` element. This issue can be fixed by ensuring that the label is accurate, descriptive, succinct, and that it is associated with the correct form control element.

3. **43 Structural elements**
- **Heading level (15) :** There are three `<h1>`, five `<h2>`, two `<h3>` and five `<h4>` elements present  .This issue can be fixed by ensuring that the text in question is truly a heading and that it is structured correctly in the page outline.
- **Unordered list (22) :** The `<ul>` element is present. This issue can be fixed by ensuring that an unordered (bulleted) list is appropriate for the context. If list items are sequential or numbered, an ordered list (`<ol>`) is likely more appropriate.
- **Navigation (3) :** A `<nav>` element or `role="navigation"` is present. This issue can be fixed by ensuring the element defines page navigation. Multiple navigation elements on one page can be differentiated with `ARIA` labels. 
- **Main content (1) :** A `<main>` element or `role="main"` is present. This issue can be fixed by ensuring the element surrounds and defines page main content.
- **Footer (2) :** A `<footer>` element or `role="contentinfo"` is present. This issue can be fixed by ensuring the element surrounds and defines page or page section footer content.

4. **12 ARIA** 
- **ARIA (5) :** An ARIA `role`, `state`, or `property` is present, excluding landmark roles, `aria-labelledby`, or `aria-describedby` which are distinct `WAVE` items. This issue can be fixed by ensuring the ARIA role, state, or property is used correctly. Use standard `HTML` accessibility features when possible. Be aware that support for ARIA is limited in older browsers and assistive technologies.
- **ARIA label (7) :** An `aria-label` or `aria-labelledby` attribute is present. This issue can be fixed by ensuring the `aria-label` value or element referenced by `aria-labelledby` provides a descriptive, accurate label. When possible, use standard `HTML <label>` or other markup to make the association.

### The HTML element to be changed

```html
1. The accessibility text of the following code is missing <button onclick="toggleFullScreen()" class="headerbtn" data-toggle="tooltip" data-placement="bottom" title="" data-original-title="Fullscreen mode">
<span class="headerbtn__icon-container">
  <i class="fas fa-expand"></i>
  </span>
</button> 
2. The <html style="--pulse-color: #fffc0230;"> is not having a lang attibute.
3. The document is containing two footer element so I will change the second footer.The code is the following:
<footer class="footer-article noprint">
<!-- Previous / next buttons-->
<div class="prev-next-area"><a class="left-prev" id="prev-link" href="conduct/faq.html" title="previous page"><i class="fas fa-angle-left"></i><div class="prev-next-info"><p class="prev-next-subtitle">
previous
</p><p class="prev-next-title">
Frequently Asked Questions
</p></div></a><a class="right-next" id="next-link" href="decision_making.html" title="next page"><div class="prev-next-info"><p class="prev-next-subtitle">
next
</p><p class="prev-next-title">
Decision-Making Guide
</p></div><i class="fas fa-angle-right"></i></a></div>
</footer>
</div></div>
<div class="footer-content row">
Footer<footer class="col footer"><p>
By The Jupyter Book community
<br></p><div class="extra_footer"><p xmlns:dct="http://purl.org/dc/terms/" xmlns:vcard="http://www.w3.org/2001/vcard-rdf/3.0#"><a rel="license" href="http://creativecommons.org/publicdomain/zero/1.0/">
Linked image with alternative text<img src="https://licensebuttons.net/p/zero/1.0/88x31.png" style="border-style: none;" alt="CC0">
</a><br>
To the extent possible under law,
<a rel="dct:publisher" href="https://jupyter.org"><span property="dct:title">
Project Jupyter
</span></a>
has waived all copyright and related or neighboring rights to
<a href="https://github.com/jupyter/governance"><span property="dct:title">
Project Jupyter Governance documents
</span></a>
. This work is published from:
<span property="vcard:Country" datatype="dct:ISO3166" content="US" about="https://jupyter.org">
United States
</span>
.</p></div><p></p></footer>
</div>
4. I am changing the landmark of the document 
<label class="overlay overlay-navbar" for="__navigation">
    <div class="visually-hidden">Toggle navigation sidebar</div>
</label>
```

### Your proposed HTML (or CSS), after change

```html
1. I will add aria-label attribute text to the make it have accessibility name like this:
<button onclick="toggleFullScreen()" class="headerbtn" data-toggle="tooltip" data-placement="bottom" aria-label="expand" title="" data-original-title="Fullscreen mode">
<span class="headerbtn__icon-container">
  <i class="fas fa-expand"></i>
  </span>
</button>
2. i will add the lang="en" attribute to the following code:
<html style="--pulse-color: #fffc0230;" lang="en">
3. I will change it to this:
<footer class="footer-article noprint">
<!-- Previous / next buttons-->
<div class="prev-next-area"><a class="left-prev" id="prev-link" href="conduct/faq.html" title="previous page"><i class="fas fa-angle-left"></i><div class="prev-next-info"><p class="prev-next-subtitle">
previous
</p><p class="prev-next-title">
Frequently Asked Questions
</p></div></a><a class="right-next" id="next-link" href="decision_making.html" title="next page"><div class="prev-next-info"><p class="prev-next-subtitle">
next
</p><p class="prev-next-title">
Decision-Making Guide
</p></div><i class="fas fa-angle-right"></i></a></div>
</div>
<section class="footer-content row">
Footer<div class="col footer"><p>
By The Jupyter Book community
<br></p><div class="extra_footer"><p xmlns:dct="http://purl.org/dc/terms/" xmlns:vcard="http://www.w3.org/2001/vcard-rdf/3.0#"><a rel="license" href="http://creativecommons.org/publicdomain/zero/1.0/">
Linked image with alternative text<img src="https://licensebuttons.net/p/zero/1.0/88x31.png" style="border-style: none;" alt="CC0">
</a><br>
To the extent possible under law,
<a rel="dct:publisher" href="https://jupyter.org"><span property="dct:title">
Project Jupyter
</span></a>
has waived all copyright and related or neighboring rights to
<a href="https://github.com/jupyter/governance"><span property="dct:title">
Project Jupyter Governance documents
</span></a>
. This work is published from:
<span property="vcard:Country" datatype="dct:ISO3166" content="US" about="https://jupyter.org">
United States
</span>
.
</p></div><p></p>
</div>
</section>
</footer>
4. I will change the landmark into <header> and put <label class="overlay overlay-navbar" for="__navigation">
    <div class="visually-hidden">Toggle navigation sidebar</div>
</label> inside the header like this:
<header>
<label class="overlay overlay-navbar" for="__navigation">
    <div class="visually-hidden">Toggle navigation sidebar</div>
</label>
</header>
5.
```

**Comment by minrk** [comment]
duplicate of #90
